### PR TITLE
ramips: add support for YunCore G720

### PR DESCRIPTION
**Pull request by dangowrt**
The YunCore G720 is a dual band 802.11ax router with 5 GbE ports.

Specs:
- SoC: MediaTek MT7621
- Ethernet: 5x GbE ports (built-in MT7530)
- Wireless 2.4GHz / 5GHz: MediaTek MT7915E
- RAM: 256MiB
- ROM: 16MiB (W25Q128)
- 1 Button (reset)
- 8 LEDs (1x system, 2x wifi, 5x switch ports)

Flash instructions:
The vendor firmware is based on OpenWrt, the sysupgrade image can be flashed using the '-F' (force) option on the CLI.
Make sure not to keep settings when doing so.

### CODE REVIEW

File: target/linux/ramips/dts/mt7621_yuncore_g720.dts [hunk 1]
```diff
@@ -0,0 +1,188 @@
+#include <dt-bindings/gpio/gpio.h>
```
Comment (musashino205) [inline target/linux/ramips/dts/mt7621_yuncore_g720.dts:1]: no `SPDX-License-Identifier` ?

File: target/linux/ramips/dts/mt7621_yuncore_g720.dts [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+	};
+};
+
+&spi0 {
+	status = "okay";
+
+	flash@0 {
+		compatible = "w25q128", "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <10000000>;
```
Comment (musashino205) [inline target/linux/ramips/dts/mt7621_yuncore_g720.dts:74]: Can't this be increased to higher frequency?

File: target/linux/ramips/dts/mt7621_yuncore_g720.dts (hunk 1 above)
Comment (dangowrt) [reply target/linux/ramips/dts/mt7621_yuncore_g720.dts:3]: Added SPDX line.

File: target/linux/ramips/dts/mt7621_yuncore_g720.dts (hunk 2 above)
Comment (dangowrt) [reply target/linux/ramips/dts/mt7621_yuncore_g720.dts]: Tested with 104 MHz limit as specified in W25Q128 datasheet, works fine.

**Comment by neheb** [comment]
@dangowrt what's s1 and s2?

**Pull request merged**